plans/build.fmf

```diff
@@ -19,3 +19,7 @@ discover:
   how: fmf
   test:
     - /build-tmt/build-image
+finish:
+  how: shell
+  script:
+    - cp -r /var/tmp/tmt/run-001 $TMT_PLAN_DATA
```

Comment:
For future readers can you add a comment here like `# move tmt test log to tmt plan data folder, the tmt integration test log can be archived in testing farm artifacts.` ?

Reply:
(not worth a whole new CI run for a comment of course, so merging now, but as part of further work)